### DISCUSSION TITLE
Foxglove on Fedora

### DISCUSSION THREAD

**Discussion by peterdavidfagan** [foxglove/community#931] [General]
I am trying to run ROS 2 on [Asahi Linux Fedora Remix](https://asahilinux.org/) such that I can debug ROS 2 code on my macbook air. I wanted to use Foxglove for visualizing and recording ROS topic data. 

I realise there isn't officially a `.rpm` release, I wished to check if anyone attempted to install with the existing `.deb` on Fedora? I plan to attempt this but opening this discussion in case others have been down this path before me.

**Comment by peterdavidfagan** [comment]
I got things running using the web UI + building foxglove bridge from source. 

![foxglove_on_mac](https://github.com/foxglove/community/assets/42982057/109f95e8-8340-43a1-a2f7-6f95c106db48)

**Reply by peterdavidfagan** [reply]
But have had trouble in installing package dependencies for popular libraries as some don't have a fedora release.

**Reply by defunctzombie** [reply]
If your plan is to use the foxglove bridge connection option for your work then your easiest approach would be to install Chrome on Fedora and use the web app. There is no difference between the web app and the desktop app for this workflow (or really for any workflow).

**Reply by peterdavidfagan** [reply]
Thanks @defunctzombie, it's working great with chrome 💯, I plan to continue with this workflow.
